packages/terra-arrange/examples/index.jsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
-import PropsTable from '../../../packages/terra-site/src/PropsTable';
-import Markdown from '../../../packages/terra-site/src/Markdown';
+import PropsTable from '../../../packages/terra-props-table/src/PropsTable';
```

Comment:
@Matt-Butler @bjankord if I change it to `terra-props-table` from fully qualified path, I get a module resolution error.